tutorial/circle.yml

```diff
@@ -0,0 +1,108 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```

Comment:
Disclaimer: I don't know what I'm talking about here.

I used this file on a fork of this repository and it ran the builder and made sure everything compiled when I triggered it manually.